test/features/errors/notfound.js

```diff
@@ -168,7 +168,6 @@ describe('404 handling', function() {
         })
         .finally(function() {
             nock.cleanAll();
-            nock.restore();
         });
```

Comment:
Apparently this was too much - we don't need to restore nock after test, because it breaks following nock tests. Cleaning all errors is enough to make everything run properly both if test pass and fail.